### DISCUSSION TITLE
converting-levels-into-xp-vice-versa/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#12] [Comments]
# converting-levels-into-xp-vice-versa/

Many games (such as my own Pixel Blacksmith and Blacksmith Slots) contain an XP / level system, where performing actions will reward experience, and eventually new levels. These new levels often unlock new content, or provide currency, so keeping players incentivised without feeling like a “grind” is a tricky balance.

https://blog.jakelee.co.uk/converting-levels-into-xp-vice-versa/

**Comment by xNewPlayer** [comment]
wow thanks. This really helped me

**Comment by SymbianMoe** [comment]
a gem <3

**Comment by ghost** [comment]
This was extremely helpful and easy to digest. Appreciate it!

**Comment by Kurtiii** [comment]
Thank you alot! <3
